### ISSUE TITLE
Use "{orgName}/{repoName}" for the slug

### ISSUE THREAD

**Issue by mdreizin** [appveyor/ci#1269]
Hello,

As member of some organizations I would like use the following expression for the slug:

```
{orgName}/{repoName}
```

For example if `orgName=foo` and `repoName=bar` then the full url will be:

https://ci.appveyor.com/project/foo/bar/branch/master

Currently as I can see CI uses the following expression:

```
{accountNameOnAppVeyor}/{projectNameOnAppVeyor}
```

https://ci.appveyor.com/project/fooUser/barProject/branch/master

One of possible solutions is to use `GitHub repository` for the slug and add a new options for example:

- [ ] Use `GitHub repository` for the slug

Travis uses `{orgName}/{repoName}` expression to build projects urls.

Do you have any plans to add something like that?

cc @FeodorFitsner

**Comment by IlyaFinkelshteyn** [comment]
Hi Marat,

We already do best effort to create slug exactly like this when people are registered with GitHub. Account name is different if person is registered specifically with email or with other source control provider account. We cannot make slug to be always like you said, because in this case we will have naming conflicts when more than one different account on AppVeyor have project which is targeted the same GitHub repository. 

Ilya.

**Comment by mdreizin** [comment]
@IlyaFinkelshteyn Thanks for the feedback, but it is not an issue to do that, because github/bitbucket/etc provides complete information about the repos and that approach is used by many CI providers like travis, circleci, drone etc.

I would suggest you to add a new option like `Use repository name for the slug`, try to generate slug based on that expression, perform validation etc. If a slug is acceptable just use it, if not - just tell user that it is already used.

IMHO it better to use auto generated URL instead of manually edited. It helps you to avoid duplicate URLs for repos, forks etc.

**Comment by ericnewton76** [comment]
Agreed.  It is easy to autogenerate the slug, then specify to the user that this isn't available. 

95%-99% of the time, the autogenerated slug will work just fine, and creates an established convention.
